modules/root/root_v2_api.rb

```diff
@@ -0,0 +1,39 @@
+class Proxy::RootV2Api < Sinatra::Base
+  helpers ::Proxy::Helpers
+
+  authorize_with_trusted_hosts
+  authorize_with_ssl_client
+
+  get "/features" do
+    begin
+
+      enabled_plugins = ::Proxy::Plugins.instance.select do |plugin|
+        plugin[:name] != :foreman_proxy && \
+          plugin[:class].ancestors.include?(::Proxy::Plugin)
```

Comment:
Interesting. Is there a case where there'd be a plugin that would not inherit ::Proxy::Plugin, or is this more of a defensive check?

Reply:
I copied it from the v1 api:
https://github.com/theforeman/smart-proxy/blob/ce744b8bee2506fe8779f561ded3977dc15feabb/modules/root/root_api.rb#L6

Reply:
This is actually copied from https://github.com/theforeman/smart-proxy/blob/develop/modules/root/root_api.rb#L6   in @ekohl's original PR.  I don't actually know the reason for it :)

Reply:
Looks like it was https://projects.theforeman.org/issues/15025 I guess it's fair enough to keep